libs/components/ag-grid/src/lib/modules/ag-grid/types/datepicker-properties.ts

```diff
@@ -1,9 +1,31 @@
 export interface SkyAgGridDatepickerProperties {
+  /**
+   * The date format for the input.
+   * @default "MM/DD/YYYY"
+   */
   dateFormat?: string;
+  /**
+   * Whether to disable the datepicker cell.
+   * @default false
+   */
   disabled?: boolean;
+  /**
+   * The latest date that is available in the calendar.
+   */
   maxDate?: Date;
+  /**
+   * The earliest date that is available in the calendar. To avoid validation errors, the time associated with the minimum date must be midnight. This is necessary because the datepicker automatically sets the time on the Date object for selected dates to midnight in the current user's time zone.
```

Comment:
```suggestion
   * The earliest date that is available in the calendar. To avoid validation errors, the time associated with the minimum date is midnight. This is necessary because the datepicker automatically sets the time on the Date object for selected dates to midnight in the current user's time zone.
```
Might just be me, but "must be" feels like we're telling consumers they need to do something. But if I'm following this, I think we handle that for them. If so, should "must be" be "is" to indicate a default instead of telling consumers something they need to do?